core/src/wallet.proto

```diff
@@ -132,6 +132,9 @@ message Key {
 
   // Encrypted version of the seed
   optional EncryptedData encrypted_deterministic_seed = 9;
+
+  // The path to the root.
+  repeated uint32 accountPath = 10 [packed=true];
```

Comment:
Shouldn't this field be optional? We have many existing wallets without this field, I guess it will throw a protobuf error if it is missing?

Reply:
I addressed this here https://github.com/bitcoinj/bitcoinj/pull/1341#issuecomment-346439720

https://developers.google.com/protocol-buffers/docs/proto

"repeated: this field can be repeated any number of times (including zero) in a well-formed message. The order of the repeated values will be preserved."